pages/apply.html

```diff
@@ -0,0 +1,380 @@
+            </li>
+            <li>
+              <input type="radio" name="entry.1263390573" value="I am not able to work full time." id="group_1263390573_2" role="radio" class="ss-q-radio" aria-required="true" required>
+              <label for="group_1263390573_2">I am not able to work full time.</label>
+            </li>
+          </ul>
+
+        </div>
+
+        <div class="usa-input">
+
+          <p><label class="ss-q-item-label" for="entry_1402744878">Select your preferred work location(s):<span class="required">*</span></label></p>
+
+          <ul class="usa-unstyled-list">
+            <li>
+              <input id="group_449208752_1" type="checkbox" name="entry.449208752" value="Chicago, IL" tabindex="0">
+              <label for="group_449208752_1">Chicago, IL</label>
+            </li>
+            <li>
+              <input id="group_449208752_2" type="checkbox" name="entry.449208752" value="New York, NY" tabindex="0">
+              <label for="group_449208752_2">New York, NY</label>
+            </li>
+            <li>
+              <input id="group_449208752_3" type="checkbox" name="entry.449208752" value="San Francisco, CA" tabindex="0">
+              <label for="group_449208752_3">San Francisco, CA</label>
+            </li>
+            <li>
+              <input id="group_449208752_4" type="checkbox" name="entry.449208752" value="Washington, D.C." tabindex="0">
+              <label for="group_449208752_4">Washington, D.C.</label>
+            </li>
+            <li>
+              <input id="group_449208752_5" type="checkbox" name="entry.449208752" value="Remote" tabindex="0">
+              <label for="group_449208752_5">Remote</label>
+            </li>
+          </ul>
+
+        </div>
+
+        <div class="usa-input">
+          <label for="entry_422263110">Where did you learn about 18F?</label>
+          <p>If someone referred you to us, please include their name.</p>
+          <textarea name="entry.422263110" rows="8" cols="0" class="ss-q-long" id="entry_422263110" dir="auto" aria-label="Where did you learn about 18F? If someone referred you to us, please include their name. "></textarea>
+        </div>
+
+
+      </fieldset>
+
+
+      <fieldset class="position">
+
+        <h2><legend>Position:</legend></h2>
+
+        <div class="usa-input">
+
+          <label class="ss-q-item-label" for="entry_227038756">What position and GS level are you applying to? <span class="required">*</span></label>
+
+          <p>GS level is the pay grade. Please see &ldquo;<a href="https://pages.18f.gov/joining-18f/pay-grades/">Government pay grades explained</a>.&rdquo;</p>
+
+          <select name="entry.227038756" id="entry_227038756" aria-label="What position and GS level are you applying to? GS level is the pay grade. Please see &quot;Government pay grades explained:&quot; https://pages.18f.gov/joining-18f/pay-grades/ " aria-required="true" required=>
+            <option value=""></option>
```

Comment:
Should GS-13 be included as well? (See also: my question on the position description.)